src/jiti.ts

```diff
@@ -286,9 +286,10 @@ export default function createJITI(
     let source = readFileSync(filename, "utf8");
 
     // Transpile
-    const isTypescript = ext === ".ts";
+    const isTypescript = ext === ".ts" || ext === ".mts";
     const isNativeModule =
       ext === ".mjs" ||
+      ext === ".mts" ||
```

Comment:
Since node.js has not native support for `.mts`, i think we shouldn't have it here